### PR TITLE
fix(integrations): Respond w/ org integration on install

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Fixes an inconsistency where we now expect more organization integration
specific details when rendering an installed integration.

max edit: 
fixes JAVASCRIPT-3V3

### CODE REVIEW

File: src/sentry/models/integration.py [hunk 1]
```diff
@@ -75,19 +75,16 @@ def add_organization(self, organization_id, default_auth_id=None, config=None):
 
         Returns True if the OrganizationIntegration was created
```
Comment (MaxBittker) [inline src/sentry/models/integration.py]: comment rot :P

File: src/sentry/models/integration.py [hunk 2]
```diff
@@ -110,12 +107,10 @@ def add_project(self, project_id, config=None):
 
         try:
             with transaction.atomic():
-                ProjectIntegration.objects.create(
+                return ProjectIntegration.objects.create(
```
Comment (MaxBittker) [inline src/sentry/models/integration.py]: this one too with the comment

File: src/sentry/integrations/pipeline.py [hunk 3]
```diff
@@ -60,7 +60,8 @@ def _finish_pipeline(self, data):
             )
         else:
             integration = ensure_integration(self.provider.key, data)
-        integration.add_organization(self.organization.id)
+
+        org_integration = integration.add_organization(self.organization.id)
```
Comment (MaxBittker) [inline src/sentry/integrations/pipeline.py]: this is serializes to a superset of the information?

**Review by MaxBittker — APPROVED** [review]
thanks for fixing this evan! would you mind fixing the comments?

File: src/sentry/integrations/pipeline.py (hunk 3 above)
Comment (EvanPurkhiser) [reply src/sentry/integrations/pipeline.py]: Yeah, if you look at the `OrganizationIntegration` serializer it actually serializes the entire Integration + configuration.

https://github.com/getsentry/sentry/blob/ba397b68677bf36088702e2adc6cb8c3c60f9f18/src/sentry/api/serializers/models/integration.py#L50-L82

**Pull request merged**